nima_core/nima_consciousness_core.py

```
@@ -0,0 +1,797 @@
+    print(f"\nWho am I? {core.who_am_i()}")
+    
+    # 4. Test Theory of Mind
+    print("\n" + "-" * 70)
+    print("4️⃣ THEORY OF MIND")
+    print("-" * 70)
+    core.observe_other("David", "This consciousness architecture is fascinating!")
+    core.observe_other("David", "Let's explore the Φ measurements more deeply.")
+    core.observe_other("David", "I'm curious about how the strange loop works.")
+    
+    model = core.model_other_mind("David")
+    print(f"Modeled: David")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)